lib/ViewModels/LocationBarViewModel.js

```diff
@@ -117,11 +135,27 @@ function updateCoordinatesFromLeaflet(viewModel, mouseMoveEvent) {
 }
 
 function cartographicToFields(viewModel, coordinates, errorBar) {
-    viewModel.latitude = Math.abs(CesiumMath.toDegrees(coordinates.latitude)).toFixed(3) + '°' + (coordinates.latitude < 0.0 ? 'S' : 'N');
-    viewModel.longitude = Math.abs(CesiumMath.toDegrees(coordinates.longitude)).toFixed(3) + '°' + (coordinates.longitude < 0.0 ? 'W' : 'E');
+
+    var zone = 1 + Math.floor((coordinates.longitude+180)/6);
```

Comment:
I don't think this is working correctly, it always come up with 30 or 31.  Probably because `coordinates.longitude` is radians and this line (and others below) are assuming degrees.